dominos/src/main/java/me/fontys/semester4/dominos/configuration/data/store/StoreImportBootstrap.java

```diff
@@ -0,0 +1,28 @@
+package me.fontys.semester4.dominos.configuration.data.store;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+import java.io.IOException;
+
+@Configuration
+public class StoreImportBootstrap {
+
+    @Value("${import.order.wildcard}")
+    private final String path = "./import-data/*.txt";
```

Comment:
Maybe just to be sure, prefix the store files with `store-` or so. Currently not necessary, but could be considered just in case